mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -121,8 +121,11 @@ def test_io_set():
     shutil.copyfile(op.join(base_dir, 'test_raw.fdt'),
                     op.join(temp_dir, 'test_one_event.fdt'))
     event_id = {eeg.event[0].type: 1}
-    read_raw_eeglab(input_fname=one_event_fname, montage=montage,
-                    event_id=event_id, preload=True)
+    test_raw = read_raw_eeglab(input_fname=one_event_fname, montage=montage,
+                               event_id=event_id, preload=True)
+
+    # test that sample indices are read python-wise (zero-based)
+    assert mne.find_events(test_raw)[0, 0] == eeg.event[0].latency - 1
```

Comment:
can you add a test to check they are all positive?

Reply:
sure, or do you prefer to throw an error **during reading** if some come out negative?

Reply:
hmm ... I would think we have a bug if something comes out negative _during reading_

Reply:
Right, I'll add a ValueError to eeglab reader if latency comes out negative. I'm wondering if it is possible though due to how eeglab handles downsampling data (latency is then represented as float, so `int(0.7)` would be 0 and then -1 after index correction). But that is an issue for a next PR (for example rounding float latencies).